src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -727,6 +736,27 @@ func (r *StatefulSetResource) getPorts() []corev1.ContainerPort {
 	return ports
 }
 
+func (r *StatefulSetResource) getStartupProbe() *corev1.Probe {
+	if r.pandaCluster.AdminAPIInternal() == nil {
+		return nil
+	}
+	return &corev1.Probe{
+		Handler: corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Host:   r.serviceFQDN,
+				Path:   "v1/status/ready",
```

Comment:
If I understand it correctly this always return HTTP 200 so I am not sure this will work